docs/installation.md

```diff
@@ -60,7 +60,7 @@ OS Specific steps are listed first, the [Common](#common) section below is neces
     sudo apt-get update
 
     # install packages
-    sudo apt install -y python3-pip python3-venv python3-pandas git
+    sudo apt install -y python3-pip python3.7-venv python3.7-dev python3-pandas git
```

Comment:
Please don't "hardcode" the version of python here.

Depending on the version of ubuntu (this is also valid for Debian), you might have python 3.8 or 3.9 installed (ubuntu 20.04 uas python 3.8 as default {source](https://wiki.ubuntu.com/FocalFossa/ReleaseNotes#Python3_by_default)).

As such, hardcoding the python-version here is absolutely wrong, as it can result in either forcing a installation of python3.7 on a system that defaults to python 3.8 (which can cause problems eventually) - or even result in a "version not available" error (not sure if it's easily possible to install python3.7 on a ubuntu 20.04 system - but there's no reason in doing so).

If it can work with `python3-dev` (i'm not a ubuntu user, so i can't really test) - otherwise this is the wrong approach to me.